site2/docs/admin-api-clusters.md

```diff
@@ -83,8 +83,8 @@ Here's an example cluster metadata initialization command:
 ```shell
 bin/pulsar initialize-cluster-metadata \
   --cluster us-west \
-  --zookeeper zk1.us-west.example.com:2181 \
-  --configuration-store zk1.us-west.example.com:2184 \
+  --metadata-store zk:zk1.us-west.example.com:2181 \
```

Comment:
it would be good to add an example not using the actual ZK root (chroot)

Reply:
Added. PTAL. @nicoloboschi